src/views/document/utils/boxes/RoutesBox.vue

```diff
@@ -27,11 +29,14 @@
 </template>
 
 <script>
+import AccordionItem from '@/components/generics/AccordionItem.vue';
 import { requireDocumentProperty } from '@/js/properties-mixins';
 
 export default {
+  components: {
```

Comment:
Thanks to https://github.com/c2corg/c2c_ui/blob/master/src/js/vue-plugins/generic-components.js, you do not need to declare the components from `generics` folder